molecule/driver/docker.py

```diff
@@ -88,6 +89,12 @@ class Docker(base.Base):
             buildargs:
                 http_proxy: http://proxy.example.com:8080/
 
+    If specifying the `CMD`_ directive in your ``Dockerfile.j2``, then you must
```

Comment:
It's also relevant if you're consuming a built image, rather than building it yourself (this is the issue I was hitting when I made the initial change)